Adds Waitangi Day and ANZAC Day to holidays.dm

cacogen opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Adds NZ's national day, Waitangi Day, as well as the Australian/New Zealand holiday that commemorates all who have served in our wars and peacekeeping operations.

I would've assumed the country was too small and unimportant for this but He Who Must Not Be Named [requested this](https://tgstation13.org/phpBB/viewtopic.php?f=2&t=27844&p=583711#p583642) and it's something I'd like to see in the game as well.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

This helps American players feel a little bit closer to New Zealand, i.e. Liberal American Heaven.

If you too are from a small, irrelevant country Americans know approximately three things about and want your holidays represented, you can try making your own pull request.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl: New Zealand
add: Added Waitangi Day and ANZAC day holidays
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

cacogen committed:
Adds Waitangi Day and ANZAC Day to holidays.dm

wesoda25 commented:
Hi reddit!

cacogen commented:
What

Megarop commented:
Do we really need even _more_ obscure holidays?

cacogen commented:
I'm of the view that the impact on you is insignificant and that the Americentrism the rest of the world has grown used to needn't be maintained in a game with an international playerbase that's set in an extranational future.

interestingusernam3 commented:
chad

Megarop commented:
> I'm of the view that the impact on you is insignificant and that the Americentrism the rest of the world has grown used to needn't be maintained in a game with an international playerbase that's set in an extranational future.

There's only like two American holidays now that Columbus day got removed (July the 4th and international hot dog day),  I'm of the view that that's not enough to warrant adding even more holidays most players don't care about to fight "americentrism".

cacogen commented:
If American holidays are underrepresented as a percentage of the imagined playerbase this PR doesn't prevent you from adding Martin Luther King Day or Presidents Day or whatever. We don't have that many holidays on the calendar to begin with. A lot of them are frivolous and unimportant (e.g. Pi Day, Talk Like a Pirate Day, Writer's Day). The impact on you is minimal (a greeting, station prefixes) so I'm not sure what the issue is. I like learning about the different holidays, personally.

jaw-sh commented:
ANZAC day is a great excuse to add more cookies for @Qustinnus's cookie cutter tool.

interestingusernam3 commented:
>Unless i'm missing one, there's only two American holidays (July the 4th and national hot dog day) now that Columbus day got removed.

The PR that properly removes it isn't merged you.

cacogen committed:
Makes the poppy the default drone hat on ANZAC day

The remembrance poppy is an artificial flower worn to commemorate those who died for their country.

wesoda25 commented:
Oh I thought this was bait

Megarop commented:
> We don't have that many holidays on the calendar to begin with.

I disagree, what I was trying to say earlier is I think we have _too many_ holidays, especially considering, like you said, most of them are obscure holidays nobody really celebrates, and adding even more holidays that nobody really celebrates just makes that problem even worse.

> The impact on you is minimal (a greeting, station prefixes) so I'm not sure what the issue is. I like learning about the different holidays, personally.

Yeah you're right i'm making it sound like a bigger deal than it actually is by writing some long wall of text about why I don't think it's worth adding

optimumtact commented:
Your appointment to FEMA should be finalized within the week. I've already discussed the matter with the Senator.

cacogen commented:
![image](https://user-images.githubusercontent.com/25089914/107047365-9bff9480-682c-11eb-8fc0-a98d5a58b274.png)

optimumtact commented:
God bless this god's own country

Pumpkinoe commented:
God defend our free land!

cacogen commented:
Honestly want to change it so that it just says Happy Waitangi day (which I think is what's supposed to go there) and so it uses timezones so it doesn't use the holiday based on server time but when Waitangi Day actually is (yesterday).

I haven't seen anyone complain about this on the servers though so they know who butters their bread.
